TACT.Net/Configs/ConfigContainer.cs

```diff
@@ -113,6 +116,43 @@ public void OpenRemote(string directory)
             LoadConfigs(directory);
         }
 
+        /// <summary>
+        /// Download and load the config files from remote
+        /// </summary>
+        /// <param name="url"></param>
+        /// <param name="directory"></param>
+        public void DownloadRemote(string url, string directory)
+        {
+            var webClient = new WebClient();
+
+            using (var cdnstream = webClient.OpenRead(String.Format("{0}/{1}/cdns", url.TrimEnd('/'), Product)))
```

Comment:
Eventually ribbit will replace http so you may want to try ribbit then fallback to this. For the interim it's fine.

Reply:
i need a local ribbit server for custom content, because the download tact storage is the first step to build a TACTHost (CASCHost 2.0)